packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -20,10 +20,14 @@ class _AccountPictures extends StatelessWidget {
     Key? key,
     this.currentAccountPicture,
     this.otherAccountsPictures,
+    this.currentAccountPictureSize,
+    this.otherAccountsPicturesSize,
   }) : super(key: key);
 
   final Widget? currentAccountPicture;
   final List<Widget>? otherAccountsPictures;
+  final double? currentAccountPictureSize;
+  final double? otherAccountsPicturesSize;
```

Comment:
What do you think about making these `Size` classes instead of doubles?

Reply:
Also, either way, they should probably default to the original values (48 and 72).

Reply:
> What do you think about making these `Size` classes instead of doubles?
What is the reason to make them classes, they are just numbers.
What do you mean by make them classes?

Reply:
> Also, either way, they should probably default to the original values (48 and 72).

I will check this.

Reply:
@justinmc 
By the way, what is the best practice to define default value, in the constructor definition or another way inside the widget where using the variables?

Reply:
Defining your default variables in the constructor is probably the most efficient way!

Reply:
@GaspardMerten
I have a basic question:
If Widget1 calls Widget2, where should I put the default values:
1. only at the Widget1 constructor?
2. only at the Widget2 constructor?
3. On both Widget1 and Widget2 constuctors?

Reply:
I would say that it depends whether or not Widget 2 is private or not. Meaning that if some user will one day directly use widget 2 then I think that yes there should be a default value in the constructor. If not, then since you are sure that Widget 2 will only be constructed by Widget 1, adding default values to Widget 2 would not bring any value. (My opinion :) )

Reply:
> What do you think about making these `Size` classes instead of doubles?

Done.

Reply:
> Also, either way, they should probably default to the original values (48 and 72).

Done.

Reply:
> I would say that it depends whether or not Widget 2 is private or not. Meaning that if some user will one day directly use widget 2 then I think that yes there should be a default value in the constructor. If not, then since you are sure that Widget 2 will only be constructed by Widget 1, adding default values to Widget 2 would not bring any value. (My opinion :) )

Indeed, makes sense.
Since the second widget is private, as you said, I put the defaults only in the first/main widget constructor which is public.